.circleci/config.yml

```diff
@@ -300,6 +300,32 @@ jobs:
       - store_test_results:
           path: /report
 
+  inshpect:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /pika
+      - run:
+          name: Running inshpect
+          command: |
+              export INSHPECT_NUMTHREADS=2
+              export INSHPECT_VERBOSE=1
+              # TODO: Currently restricted to libs, remove libs when regular inspect tool is removed
```

Comment:
Just to clarify, this TODO will stay for now. I will keep #6 open after this is merged.